homeassistant/components/fritzbox/sensor.py

```diff
@@ -159,6 +159,7 @@ def value_scheduled_preset(device: FritzhomeDevice) -> str:
         translation_key="comfort_temperature",
         native_unit_of_measurement=UnitOfTemperature.CELSIUS,
         device_class=SensorDeviceClass.TEMPERATURE,
+        entity_category=EntityCategory.CONFIG,
```

Comment:
Sensors cannot be config items, as they don't have input.

Reply:
mhhh ... sounds reasonable - so I assume diagnostic would be the correct category for entities which shows a configuration value (_not a measured one_) ?